pkg/oci/skopeo.go

```diff
@@ -32,8 +32,14 @@ func LoadPodman(dir, imageName string) error {
 }
 
 // Push image to registry
-func Push(dir, imageName string) error {
-	cmd := exec.Command("nix", "run", "nixpkgs#skopeo", "--", "copy", "--insecure-policy", "dir:"+dir, "docker://"+imageName)
+func Push(dir, imageName string, digest bool, auth string) error {
+	var cmd *exec.Cmd
+	if digest {
+		cmd = exec.Command("skopeo", "copy", "--insecure-policy", "dir:"+dir, "docker://"+imageName+"@@unknown-digest@@", "--digestfile=bsf-result/digest", "--dest-creds", auth)
```

Comment:
We can't rely on the directory name being `bsf-result`. It is configurable by the user.

Reply:
Can we use `dir` argument somehow?